pkg/ckecli/cmd/reboot_queue_cancel_all.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/cybozu-go/cke"
+	"github.com/cybozu-go/well"
+	"github.com/spf13/cobra"
+)
+
+var rebootQueueCancelAllCmd = &cobra.Command{
+	Use:   "cancel-all",
+	Short: "cancel all the reboot queue entries",
+	Long:  `Cancel all the reboot queue entries.`,
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		well.Go(func(ctx context.Context) error {
+			entries, err := storage.GetRebootsEntries(ctx)
+			if err != nil {
+				return err
+			}
+
+			for _, entry := range entries {
+				if entry.Status == cke.RebootStatusCancelled {
+					continue
+				}
+
+				entry.Status = cke.RebootStatusCancelled
+				err := storage.UpdateRebootsEntry(ctx, entry)
+				if err != nil {
```

Comment:
`ErrNotFound` would be returned if the queue entry has just finished.  Though we can retry `ckecli rq cancel-all`, it would be good to ignore `ErrNotFound` here.